### DISCUSSION TITLE
Add ability to add headers to webhook requests

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#4703] [Feature Requests]
It would be useful to have the ability to add headers to webhook requests.

My use case:

Using DroneCI to build my site with Gridome and using Directus. The issue here is I would like an update in directus to trigger a new build BUT it looks like at the moment Drone requires someone to be authenticated before making the request using bearer authentication with a token.

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/3649 and https://github.com/directus/directus/discussions/3000
